packages/extension-bubble-menu/src/bubble-menu-plugin.ts

```diff
@@ -257,6 +266,7 @@ export class BubbleMenuView implements PluginView {
     this.updateDelay = updateDelay
     this.resizeDelay = resizeDelay
     this.appendTo = appendTo
+    this.scrollTarget = options?.scrollTarget ?? null
```

Comment:
Possible refactor: 

```suggestion
    this.scrollTarget = options?.scrollTarget ?? window
```

Does it make sense or would it break things?